### DISCUSSION TITLE
I lost all private items how can I recover it please suggest me

### DISCUSSION THREAD

**Discussion by Ajay787462** [d4rken-org/sdmaid-se#1398] [Q&A]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Smartphone (please complete the following information):**
 - Device: [e.g. Pixel 9]
 - OS: [e.g. Android 17]
 - Root status [e.g. Rooted - Magisk v25.2]
 - SD Maid 2/SE version [e.g. v0.6.6-rc6]

**Debug log**
If you can reproduce the bug, record the event in [a debug log](https://github.com/d4rken-org/sdmaid-se/wiki/Bugs#debug-log).

**Screenshots/Video**
If applicable, add screenshots or a video if it helps to explain your problem.

**Additional context**

**Comment by MarkyMarkDE** [comment]
Are you sure, that your topic is right here?
You can find lot of data recovery apps in store but they are really expensive and no guarantee, that the data can be restored. Sorry can't suggest a special one for this, every user know that important data should be backed up on multiple ways (this is basic knowledge since MS DOS) e.g saved in cloud, on computer or on a second phone or whatever. No computer in the world is free of failures and data loss. I'm really sorry for you, now you learned it by the hard way.
